backend-rust/docker-compose.yml

```diff
@@ -9,4 +9,4 @@ services:
       - ./data:/var/lib/postgresql/data
     environment:
       POSTGRES_PASSWORD: ${PGPASSWORD}
-      POSTGRES_DB: ccd-scan
+      POSTGRES_DB: ccdscan
```

Comment:
Not really that relevant, but I am doing pg_dump and restore and then it helps with naming the database the same